docs/dev.md

```diff
@@ -4,6 +4,8 @@
 
 Start a Liberty server in dev mode. This goal also invokes the `create`, `install-feature`, and `deploy` goals before starting the server. **Note:** This goal is designed to be executed directly from the Maven command line.
 
+N.B. starting in the tech preview in the 3.5.2-SNAPSHOT dev mode also invokes `generate-features`. See [generate-features](generate-features.md) for details.
```

Comment:
Consider also adding `recompileDependencies` to the parameter section: https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md?plain=1#L84-L99

Reply:
Before changing the dev documentation I'd like to keep the changes localized as YK suggested and just add the `generateFeatures` parameter to the dev mode part of the generate-features.md.